Lib/test/test_xml_etree.py

```diff
@@ -2680,18 +2681,101 @@ class Y(X, ET.Element):
         e = ET.Element('foo')
         e.extend(L)
 
-    def test_remove_with_mutating(self):
-        class X(ET.Element):
+    def test_remove_with_clear(self):
+        # See: https://github.com/python/cpython/issues/126033
+
+        # Until the discrepency between "del root[:]" and "root.clear()" is
+        # resolved, we need to keep two tests. Previously, using "del root[:]"
+        # did not crash with the reproducer of gh-126033 while "root.clear()"
+        # did.
+
+        E = ET.Element
+
+        def test_remove(root, target, raises):
```

Comment:
BTW, keeping a reference to the removed element can matter. `test_remove()` adds a reference (as function parameter). I think that it is better to inline it.